.env

```diff
@@ -0,0 +1,8 @@
+## Firebase
```

Comment:
We need to remove .env files from the repo. This is a security risk because of the api keys and stuff that we throw in there.